src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala

```diff
@@ -296,22 +292,22 @@ class TezosPlatformDiscoveryOperations(
     * @param  tableName name of the table from which we extract attributes
     * @return list of attributes as a Future
     */
-  override def getTableAttributes(tableName: String): Future[Option[List[Attribute]]] =
+  override def getTableAttributes(entityPath: EntityPath): Future[Option[List[Attribute]]] =
```

Comment:
Why the change from table name to EntityPath

Reply:
because extracting metadata configuration for given entity requires full EntityPath, and before there was no information passed about platform/network to get information about given entity